app/services/bgs/dependent_service.rb

```diff
@@ -51,12 +51,26 @@ def submit_686c_form(claim)
       # This is because we are currently relying on the presence of a PDF and
       # absence of a BGS-established claim to identify cases where Form 686c-674
       # submission failed.
-      VBMS::SubmitDependentsPdfJob.perform_async(
+      vet_info = KmsEncrypted::Box.new.encrypt(form_hash_686c.to_json)
```

Comment:
setting vet_info inside the flipper as well. I believe you can delete this line.